app/src/main/java/fr/gaulupeau/apps/Poche/ui/ReadArticleActivity.java

```diff
@@ -785,6 +787,17 @@ public void onClick(DialogInterface dialog, int which) {
                             case 1:
                                 ServiceHelper.addLink(ReadArticleActivity.this, url);
                                 break;
+                            case 2:
```

Comment:
can you please adapt (rename, add parameter) and re-use the function "private void copyOriginalURL()" in this case?

Reply:
Updated it.

Reply:
`copyOriginalURL()` only copies the article's URL to clipboard, not as you wanted, the URL of the link you clicked. Can you please adapt the method (rename, add parameter for URL to copy)?

Reply:
I tried to do it right this time.